postgis/lwgeom_functions_basic.c

```diff
@@ -1546,12 +1546,12 @@ Datum LWGEOM_makepoly(PG_FUNCTION_ARGS)
 		holes = lwalloc(sizeof(LWLINE *) * nholes);
 		for (i = 0; i < nholes; i++)
 		{
-#if GCC_VERSION >= 40604
```

Comment:
If you're stripping out use of `GCC_VERSION` (which is fine, I just am too dim to do the major/minor/patch logic), can you also remove the definition from postgis_config.h.in?

Reply:
I don't see any `GCC_VERSION` in my `postgis_config.h.in` or anywhere in the tree. Did you add it and forgot to push it?